packages/cli/src/commands/template/buildWithProxy.ts

```diff
@@ -0,0 +1,210 @@
+import child_process from 'child_process'
+import * as e2b from 'e2b'
+import * as http from 'http'
+import * as url from 'url'
+import * as https from 'node:https'
+import { confirm } from '../../utils/confirm'
+import { USER_CONFIG_PATH, UserConfig } from 'src/user'
+import * as fs from 'fs'
+
+const PORT = 49984
+
+export async function buildWithProxy(
+  userConfig: UserConfig | null,
+  connectionConfig: e2b.ConnectionConfig,
+  accessToken: string,
+  template: { templateID: string; buildID: string },
+  root: string
+) {
+  if (!userConfig?.dockerProxySet) {
+    console.log(
+      "There was an issue with Docker authentication. Here's a temporary workaround: https://e2b.dev/docs/cli/build-auth-error. Once you've followed the guide, please continue."
+    )
+    const yes = await confirm('Did you complete the guide?')
```

Comment:
Have you completed the steps from the https://e2b.dev/docs/cli/build-auth-error workaround guide?

Reply:
```suggestion
    const yes = await confirm('Have you completed the steps from the https://e2b.dev/docs/cli/build-auth-error workaround guide?')
```